app/pages/foreign/ForeignPropertySummaryPage.scala

```diff
@@ -127,106 +127,106 @@ case class ForeignSummaryPage(foreignCYADiversionService: ForeignCYADiversionSer
-                Some(countryCode)
-              ) {
-                controllers.foreign.adjustments.routes.ForeignAdjustmentsStartController
-                  .onPageLoad(taxYear, countryCode, isPIA = true)
-              },
-              taskListTagForAdjustments,
-              s"foreign_property_adjustments_$countryCode"
-            )
-          )
-        )
-      case Some(false) =>
-        taskList.appendedAll(
-          Seq(
-            TaskListItem(
-              "summary.expenses",
-              foreignCYADiversionService.redirectCallToCYAIfFinished(
-                taxYear,
-                userAnswers,
-                ForeignCYADiversionService.EXPENSES,
-                Some(countryCode)
-              ) {
-                controllers.foreign.expenses.routes.ForeignPropertyExpensesStartController
-                  .onPageLoad(taxYear, countryCode)
-              },
-              taskListTagForExpenses,
-              s"foreign_property_expenses_$countryCode"
-            ),
-            TaskListItem(
-              "summary.allowances",
-              foreignCYADiversionService.redirectCallToCYAIfFinished(
-                taxYear,
-                userAnswers,
-                ForeignCYADiversionService.ALLOWANCES,
-                Some(countryCode)
-              ) {
-                controllers.foreign.allowances.routes.ForeignPropertyAllowancesStartController
-                  .onPageLoad(taxYear, countryCode)
-              },
-              taskListTagForAllowances,
-              s"foreign_property_allowances_$countryCode"
-            ),
-            TaskListItem(
-              "summary.structuresAndBuildingAllowance",
-              getSbaRouteDestination(taxYear, countryCode, userAnswers, taskListTagForSba),
-              taskListTagForSba,
-              s"foreign_structure_and_building_allowance_$countryCode"
-            ),
-            TaskListItem(
-              "summary.adjustments",
-              foreignCYADiversionService.redirectCallToCYAIfFinished(
-                taxYear,
-                userAnswers,
-                ForeignCYADiversionService.ADJUSTMENTS,
-                Some(countryCode)
-              ) {
-                controllers.foreign.adjustments.routes.ForeignAdjustmentsStartController
-                  .onPageLoad(taxYear, countryCode, isPIA = false)
-              },
-              taskListTagForAdjustments,
-              s"foreign_property_adjustments_$countryCode"
-            )
-          )
-        )
-      case None => taskList
+      case Some(true)  => Seq(foreignTaxTaskList, foreignIncomeTaskList, claimingAdjustmentsTaskList)
+      case Some(false) if accrualsOrCash => Seq(foreignTaxTaskList, foreignIncomeTaskList, expensesTaskList, allowancesTaskList) ++ Seq(sbaTaskList) ++ Seq(nonClaimingAdjustmentsTaskList)
+      case Some(false) => Seq(foreignTaxTaskList, foreignIncomeTaskList, expensesTaskList, allowancesTaskList) ++ Seq(nonClaimingAdjustmentsTaskList)
```

Comment:
this is how you use the copy function

```suggestion
         adjustmentsTaskList.copy(call =
              foreignCYADiversionService.redirectCallToCYAIfFinished(
                taxYear,
                userAnswers,
                ForeignCYADiversionService.ADJUSTMENTS,
                Some(countryCode)
              ) {
                controllers.foreign.adjustments.routes.ForeignAdjustmentsStartController
                  .onPageLoad(taxYear, countryCode, isPIA = false)
              }
            )
```